WORKSPACE

```diff
@@ -12,10 +12,7 @@ http_archive(
 http_archive(
     name = "org_tensorflow",
     patch_args = ["-p1"],
-    patches = [
-        # Fix tensorflow not being able to read image files on Windows
-        "//:flutter/third_party/tensorflow-fix-file-opening-mode-for-Windows.patch",
```

Comment:
This patch is still required for the Flutter app

Reply:
Fixed, thanks.

Reply:
Yes, please keep this windows patch.

And maybe make this a 1.1 only patch so that we don't have to remove it when TF 2.7 or later is used to replace 2.6.

Reply:
How do I make it 1.1 only?

Reply:
@d-uzlov and @anhappdev please create a new 1.1 branch (supposedly based on the branch you used to created the intergrated 1.1 apk)

Reply:
I created a new branch 'release-1.1-Jan-2022'